Notebooks/Simulations/skypy_lenstronomy.ipynb

```diff
@@ -29,7 +29,7 @@
      "name": "stdout",
```

Comment:
<p>I spotted a typo 'th g,r,i'...</p><p> </p><p>Perhaps we can add some more links to the SimulationAPI classes</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lenstronomy/lenstronomy-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>